### PR TITLE
Gh action for release mgmt

### PR DESCRIPTION
**Pull request by mkuthan**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: Build
 
-on: [push, pull_request]
+on:
```
Comment (adrianmroz) [inline .github/workflows/build.yml:3]: 👍

File: .release-it.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "github": {
```
Comment (adrianmroz) [inline .release-it.json:2]: Co to robi?

File: .release-it.json [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+    "release": true
```
Comment (adrianmroz) [inline .release-it.json:3]: What is that for?

**Review by adrianmroz — APPROVED** [review]

File: .release-it.json (hunk 3 above)
Comment (mkuthan) [reply .release-it.json:3]: instead of command line option

Comment (mkuthan) [reply .release-it.json:3]: by default GitHub releases are disabled

**Pull request merged**